### ISSUE TITLE
[request] Include a zsh completion file like pacman does

### ISSUE THREAD

**Issue by txtsd** [archlinuxfr/yaourt#178]
Pacman: https://projects.archlinux.org/pacman.git/tree/contrib/zsh_completion.in
There's an [unofficial one for yaourt](https://github.com/zsh-users/zsh-completions/blob/master/src/_yaourt), but it is 2 years old and doesn't function well like pacman's.
It would be nice to have official shell completion!

**Comment by Skunnyk** [comment]
Yup, I would happily add a zsh completion file is someone pull request it :)

**Comment by nullptrT** [comment]
A zsh-completion script was added to zsh-completions last year:
https://github.com/zsh-users/zsh-completions/blob/master/src/_yaourt
Thanks to the contributor! :)
